### PR TITLE
feat: Apl-10 ingress apps update

### PR DESCRIPTION
**Pull request by Ani1357**
This PR updates the following helm charts:
- Cert-Manager
  - Chart: `v1.11.4 -> v1.16.1`
  - App: `v0.13.4 -> v0.15.0`
- External DNS
  - Chart: `6.20.4 -> 8.3.9`
  - App: `0.13.4 -> 0.15.0`
- Ingress-Nginx
  - Chart: `4.6.1 -> 4.11.2`
  - App: `1.7.1 -> 1.11.2`
- Oauth2-Proxy
  - Chart: `3.7.4 -> 7.7.24`
  - App: `7.4.0 -> 7.7.0`

What was tested:
- Creating and exposing APL service to the public: ✅
- Creating and exposing APL using a CNAME [With TLS Termination at the NGINX Controller](https://apl-docs.net/docs/for-devs/console/services#with-tls-termination-at-the-nginx-controller): ✅
- Creating and exposing APL using a CNAME [With TLS Termination at the Application (Pod) Level](https://apl-docs.net/docs/for-devs/console/services#with-tls-termination-at-the-application-pod-level) : 🚫 (created bug [APL-287](https://jira.linode.com/browse/APL-287))
- Creating and exposing APL using the Has Certificate option:  🚫(will be removed)

### CODE REVIEW

File: values/external-dns/external-dns.gotmpl
Comment (srodenhuis) [inline values/external-dns/external-dns.gotmpl:1]: Not part of this PR, buy I don't see a need to manage the tag in the .gotmpl. Why not use the chart default?

**Comment by srodenhuis** [comment]
Tested the upgrade and went smooth.

**Review by srodenhuis — APPROVED** [review]
LGTM

File: values/external-dns/external-dns.gotmpl
Comment (Ani1357) [reply values/external-dns/external-dns.gotmpl:1]: I agree, updated accordingly.
